### ISSUE TITLE
Scroll Limitation.

### ISSUE THREAD

**Issue by RichardSummers** [IntelligentVisibility/filmStripImageBrowser#3]
When using the scrollbar only,  prevent the filmstrip scrolling too far.

Example / Description:
When scrolling to the end of the filmstrip, the very last image displayed is shown at the extreme left side, with empty white space taking up the remainder of the filmstrip.

Maybe the display canvas could be set to only scroll far enough for the very last image to be on the extreme right.

Hope that made sense.

**Comment by IntelligentVisibility** [comment]
So essentially only enable scrolling if you have more images on the canvas then the canvas/window width has?

**Comment by IntelligentVisibility** [comment]
Yes this scrollbar max calculates everytime you drop an image and is measured by all images.left + thumbNailW

**Comment by IntelligentVisibility** [comment]
Might need to GTM.
:)

**Comment by RichardSummers** [comment]
We can GTM whenever you have time. Just let me know in advance, so I can ensure I am at home.

**Comment by RichardSummers** [comment]
What I was trying to say is that with the scrollbar, you can scroll way past the last image, so you end up with a large amount of white space at the end.

Maybe we need to calculate the width of each image in the filmstrip, multiplied by the number of images, plus say 20 pixels spacing between each image - then make this total width the maximum that the scrollbar can scroll to?

Just an idea.

**Comment by RichardSummers** [comment]
I just opened a folder containing 103 images, but the scrollbar (when at maximum), doesn't go far enough. If I make the window wider, I can see more of the images.

**Comment by IntelligentVisibility** [comment]
I already am doing what you suggested above :)

'// CALCULATE SCROLL BAR MAXIMUM
Dim thisMaximumW as Integer = (cvsScroller.pictureArray.Ubound +1) * (cvsScroller.thisScaledW)
Return thisMaximumW'

**Comment by RichardSummers** [comment]
Hmm, I have no idea then?
There seems to be 2 diametrically opposed issues:

1) With a relatively low number of images - the scrollbar scrolls way too far.
2) With a large number of images - the scrollbar does not scroll far enough.

This is way beyond me :)

**Comment by IntelligentVisibility** [comment]
Are you also sure that you are using images that don't have a large transparent border thus giving us the appearance of issues?

**Comment by RichardSummers** [comment]
Definitely not - I do not even have 1 image with any transparency.
